### PR TITLE
Example plugins

### PR DESCRIPTION
**Pull request by hazmat345**
This points the default plugins directory in bartender at the example-plugins repo. This removes the burden of maintaining separate example plugins specifically for bartender.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "plugins"]
+	path = plugins
+	url = ../example-plugins
```
Comment (loganasherjones) [inline .gitmodules]: Sorry, I might be missing something. Shouldn't this be pointing at a repo?

Comment (hazmat345) [reply .gitmodules]: The relative path is what's pointing at a repo. It's easier (and better) than using github.com/ urls.

Comment (hazmat345) [reply .gitmodules]: Sorry, I should probably have phrased that better. You can use relative paths when specifying submodules for repos in the same group. That way we don't have to hardcode https://github.com/beer-garden/example-plugins, we can just rely on `example-plugins` being a sibling of `bartender` in the `beer-garden` group.

Comment (loganasherjones) [reply .gitmodules]: Ah okay cool. Thanks

**Comment by loganasherjones** [comment]
:+1:

**Pull request merged**